src/modules/transaction/views/ViewTransaction.vue

```diff
@@ -95,15 +102,33 @@ const loadTxn = async () => {
   if (!props.hash) {
     return;
   }
+  const data = sessionStorage.getItem("storeFailedTxnDetails")
+    if(data){
+      let getFailedTxns: Array<{networkName:string, txnHash:string, errMsg:string}> = JSON.parse(data)
+      failedTxnDetails.value = getFailedTxns
+    }
+
+  failedTxnDetails.value.forEach((value) =>{
+    if(networkName.value === value.networkName && props.hash === value.txnHash){
+      failedStatus.value = value.errMsg
+      isTxnFailed.value = true
```

Comment:
should return after finding the data, but forEach cannot do a return exit, so maybe change to find ?